packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -2354,9 +2354,13 @@ export class StripeHelper extends StripeHelperBase {
     const productMetadata = this.mergeMetadata(plan, abbrevProduct);
     const {
       emailIconURL: planEmailIconURL = '',
-      downloadURL: planDownloadURL = '',
+      downloadURL,
```

Comment:
Why is this still here since we are replacing the "download" url in this patch?

Reply:
Ok, I see a TODO below that's likely the reason for this.  Is there a jira file to remove this?